db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js

```diff
@@ -778,6 +905,7 @@ VALUES
 	('Nokia 2G parameters','Nokia 2G parameters', $$${NOKIA_2G_KEY_PARAMAETERS}$$, '{}', 'table',1, true),
 	('Nokia 3G parameters','Nokia 3G parameters', $$${NOKIA_3G_KEY_PARAMAETERS}$$, '{}', 'table',1, true),
 	('Nokia 4G parameters','Nokia 4G parameters', $$${NOKIA_4G_KEY_PARAMAETERS}$$, '{}', 'table',1, true),
+	('COMBINED KEY PARAMAETERS','COMBINED KEY PARAMAETERS', $$${COMBINED_KEY_PARAMAETERS}$$, '{}', 'table',1, true),
```

Comment:
Can you change the report name to **Combined Key Parameters**

Reply:
Changed